### ISSUE TITLE
Backport:Project owner/creator enable project monitoring not success

### ISSUE THREAD

**Issue by loganhz** [rancher/rancher#20020]
https://github.com/rancher/rancher/issues/19381

**Comment by alena1108** [comment]
can be tested with rc7

**Comment by jiaqiluo** [comment]
The validation fails because of the same as https://github.com/rancher/rancher/issues/19381#issuecomment-490191909

```
2019/05/07 20:16:34 [ERROR] AppController p-6dtkd/project-monitoring [helm-controller] failed with : 
catalogtemplateversions.management.cattle.io 
"system-library-rancher-monitoring-0.0.3" not found
```

**Comment by jiaqiluo** [comment]
The working around here is to set the branch to `dev` for the `system-library`

--------

The bug fix is validated on Rancher: v2.2.3-rc7

Step:
- as the admin, provision an EC2 cluster 
- as the admin, create a standard user and set it as the cluster member
- login as the new user and create a project.
- enable the project monitoring in that project.

Result:
- the project monitoring is up 
- no errors showing in the log

**Comment by loganhz** [comment]
please change the system-library branch to dev manually
